client/components/dot-pager/index.jsx

```diff
@@ -58,18 +60,20 @@ const Controls = ( { currentPage, numberOfPages, setCurrentPage } ) => {
 					aria-label={ translate( 'Next' ) }
 					onClick={ () => setCurrentPage( currentPage + 1 ) }
 				>
+					{ showControlLabels && translate( 'Next' ) }
 					<Icon icon={ arrowRight } size={ 18 } fill="currentColor" />
 				</button>
 			</li>
 		</ul>
 	);
 };
 
-export const DotPager = ( { children, className } ) => {
+export const DotPager = ( { showControlLabels = false, children, className, ...props } ) => {
 	const [ currentPage, setCurrentPage ] = useState( 0 );
 	return (
-		<Card className={ className }>
+		<div className={ className } { ...props }>
```

Comment:
I only just saw that we could remove the explicit `className` prop and roll it into the `...props` thingy. Not that it matters.

Reply:
Yeah, I've sort of muddled the pattern a bit, but I do prefer to call out all the properties the component wants and additionally signal that it's extensible. I actually find it kind of weird that it's not  the universal default, but we can get away with naming every property when we're the only ones using the components, and anyone who wants to use it can change the source.

What's odd is that in the devdocs example we set `readmeFilePath` on the `DotPager`, but before I added this we just throw that away.

Anyway, I'll leave this unless you actively want me to change it.

Reply:
No problem 🚢